interface/src/app/greenelab/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * This module includes a few utility components that will be shared in other
+ * modules to avoid duplicate codes.
+ */
+
+angular.module('greenelab.utils', [])
+.constant('ApiBasePath', '/api/v0/')
```

Comment:
I think the reason for the confusion here is that we have two distinct entities: `ApiBasePath` and `errGen`. Both of them are useful to have at a globally reusable level, but I think they each belong in different files.

`ApiBasePath` is indeed a constant, and it could go into a new `app/constants.js` file as suggested above. For `errGen`, I like `app/utils.js`. Although each file will be small to start, we can use these new "homes" for similar entities that will get created down the road.

I'm also happy with just keeping them both in `app/utils.js` for now and splitting out the constants when that file grows.